data/actions/scripts/tools/fishing.lua

```diff
@@ -1,20 +1,93 @@
-local waterIds = {493, 4608, 4609, 4610, 4611, 4612, 4613, 4614, 4615, 4616, 4617, 4618, 4619, 4620, 4621, 4622, 4623, 4624, 4625, 7236, 10499, 15401, 15402}
+-- Water Ids + 1 no fish -1 with fish
+local waterWithFish = {4608, 4610, 4612, 4614, 4616, 4618, 4664, 7236, 15401}
+local waterWithoutFish = {4609, 4611, 4613, 4615, 4617, 4619, 4665, 7237, 15402}
+local waterSpiritId = 10449
```

Comment:
sprite?

Reply:
Water spirit, there was code already in the fishing script to allow fishing in a water spirit. I guess you can get some type of rewards.

Reply:
Water spirit corpse*

Reply:
its called water elemental, not water spirit, no?